scripts/core/editor3/components/suggestions/SuggestionPopup.jsx

```diff
@@ -56,7 +72,7 @@ export class SuggestionPopup extends Component {
         };
 
         return (
-            <Dropdown key={this.props.keyForDropdown} open={true}>
```

Comment:
This is required so before [author data is fetched](https://github.com/superdesk/superdesk-client-core/pull/2079/files#diff-4b54af773b8aaa818f599e3086f73861R33) there is no empty box displayed.